### ISSUE TITLE
Add support for restartable Tailers.

### ISSUE THREAD

**Issue by peter-lawrey** [OpenHFT/Chronicle-Queue#562]
Add a way for the Tailer to store where it is up to and continue when a process restarts.

```java
    /**
     * <b>
     * Tailers are NOT thread-safe, sharing the Tailer between threads will lead to errors and unpredictable behaviour.
     * </b>
     *
     * @param id unique id for a tailer which uses to track where it was up to
     * @return a new ExcerptTailer to read sequentially.
     */
    @NotNull
    ExcerptTailer createTailer(String id);
````

**Comment by andyliukun** [comment]
I am looking for the restartable tailers too.

**Comment by dpisklov** [comment]
This is already available in latest version

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.10](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.10), [BOM-2.17.196](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.196)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.10](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.10), [BOM-2.17.225](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.225)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.10](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.10), [BOM-2.17.241](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.241)
